### PR TITLE
Build fuzzers in CI

### PR DESCRIPTION
**Pull request by nikitamikhaylov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Not for changelog (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

Detailed description / Documentation draft:
My first try to build them with clang-tidy was failed. Compiler simply segfaults. So, try to build them with sanitizers.

### CODE REVIEW

**Comment by nikitamikhaylov** [comment]
@Mergifyio update

**Comment by nikitamikhaylov** [comment]
@Mergifyio update

File: docker/packager/binary/build.sh [hunk 1]
```diff
@@ -35,6 +35,22 @@ ninja $NINJA_FLAGS clickhouse-bundle
 ccache --show-config ||:
 ccache --show-stats ||:
 
+# Also build fuzzers if any
+FUZZER_TARGETS=$(find ../src -name '*_fuzzer.cpp' -execdir basename {} .cpp ';' | tr '\n' ' ')
+
+mkdir -p /output/fuzzers
```
Comment (alesapin) [inline docker/packager/binary/build.sh:41]: Probably we need the same changes for `deb/build.sh`.

Comment (nikitamikhaylov) [reply docker/packager/binary/build.sh:41]: Sure

**Comment by nikitamikhaylov** [comment]
I think that linking with libfuzzer somehow affects the clickhouse binary itself. You can see weird failures with TSan in previous commit.

**Comment by nikitamikhaylov** [comment]
@Mergifyio update

File: docker/packager/packager [hunk 2]
```diff
@@ -105,6 +105,9 @@ def parse_env_variables(build_type, compiler, sanitizer, package_type, image_typ
     if image_type == "deb" or image_type == "unbundled":
         result.append("DEB_CC={}".format(cc))
         result.append("DEB_CXX={}".format(cxx))
+        # For building fuzzers
```
Comment (alesapin) [inline docker/packager/packager:108]: Correct

**Review by alesapin — APPROVED** [review]
LGTM

**Pull request merged**